### PR TITLE
improve exit message

### PR DESCRIPTION
**Pull request by DidierRLopes**
Notice the "the" 😜

image

### CODE REVIEW

File: openbb_terminal/terminal_helper.py [hunk 1]
```diff
@@ -45,23 +45,31 @@ def print_goodbye():
     # "...when offered a flight to the moon, nobody asks about what seat."
 
     console.print(
-        "OpenBB Terminal is the result of a strong community building an "
-        "[param]investment research platform for everyone.[/param]\n\n"
-        "Join us on [cmds]https://openbb.co/discord[/cmds], "
-        "show your appreciation on [cmds]https://twitter.com/openbb_finance[/cmds],\n"
-        "ask support on [cmds]https://openbb.co/support[/cmds], "
-        "or even request a feature on [cmds]https://openbb.co/request-a-feature[/cmds]\n"
+        "[param]OpenBB Terminal is the result of a strong community building an "
```
Comment (jmaslek) [inline openbb_terminal/terminal_helper.py:48]: *the :P

Comment (DidierRLopes) [reply openbb_terminal/terminal_helper.py]: check my new commit 🤣

**Review by jmaslek — APPROVED** [review]
u are an exit message

**Comment by JerBouma** [comment]
Wouldn't it be better to add a generic recruiting message? If people now continue to use this version of the terminal, they constantly see there is a vacancy open for UI/UX while perhaps in a week or two it isn't there anymore.

**Comment by DidierRLopes** [comment]
@JerBouma good point, just updated to reflect that

**Pull request merged**